demisto_sdk/commands/common/tools.py

```diff
@@ -1904,20 +1911,24 @@ def get_release_note_entries(version='') -> list:
 
 
 def get_current_usecases() -> list:
-    """Gets approved list of usecases from current branch
+    """Gets approved list of usecases from current branch (only in content repo).
 
     Returns:
         List of approved usecases from current branch
     """
-    approved_usecases_json, _ = get_dict_from_file('Tests/Marketplace/approved_usecases.json')
-    return approved_usecases_json.get('approved_list', [])
+    if not is_external_repository():
+        approved_usecases_json, _ = get_dict_from_file('Tests/Marketplace/approved_usecases.json')
+        return approved_usecases_json.get('approved_list', [])
+    return []
```

Comment:
this will cause some validations errors.
for non-content repositories, we should fetch the file from the master branch in the content repo (see `get_approved_usecases`).

same for the `approved_tags` file

Reply:
it does fetch from content master, and adds to it all the use cases/tags from the current branch (if a content branch) - https://github.com/demisto/demisto-sdk/pull/1364/files